code/modules/mob/living/carbon/human/species_types/roguetown/human/humen.dm

```diff
@@ -5,13 +5,13 @@
 	name = "Humen"
 	id = "humen"
 	desc = "<b>Humen</b><br>\
-	Humen (or Human) are the eldest of the weeping gods creation. Noted for their\
-	tenacity and overwhelming population, humans tend to outnumber the other races. \
-	at a rate of about ten to one in regions such as Grenzelhoft. Althrough to the west \
-	the opposite is true. Humen come from a vast swathe of cultures and ethnicity, most of which\
-	have historically been at odds with one another. Being the eldest of the weeping God, humen\
+	Humens (or \"Humans\") are the eldest of the Weeping God's creations. Noted for their\
+	tenacity and overwhelming population, humens are the most commonly seen race across the lands, \
+	at a rate of about ten to one in regions such as Grenzelhoft. However, to the west \
+	the opposite is true. Humens come from a vast swathe of cultures and ethnicities, most of which\
+	have historically been at odds with one another. Being the eldest creations of the Weeping God, humens\
 	tend to find fortune easier than the other races, and are so diverse that no other racial trait\
-	are dominant in their species... \
+	are dominant in their species.<br>\
```

Comment:
"...no other racial *traits* are dominant..." or "...no other racial trait *is* dominant..."

Reply:
... Shoot